### DISCUSSION TITLE
Q-Stepper modifications

### DISCUSSION THREAD

**Discussion by mtn090899** [quasarframework/quasar#13585] [General - Components / Directives / etc] [answered]
Hi!

I actually asked a question here not too long ago, and fell deeper in the rabbit hole. I have two questions.

1. Is it possible to change css of pieces within the component dynamically? Such as having the grey bar in the stepper turn green after the step before it is passed, or having an active step appear bigger than the other steps.
2. Is it possible to change the spacing of the grey bar so that it can be more center and touching the steps? I'm not quite sure how to go about that. 

This is what I have as of now: 

![image](https://user-images.githubusercontent.com/78068147/171434378-aa7cb99f-fcef-4267-bb7c-7f341d7b2af5.png)

**Comment by suleimanas** [comment, marked as answer]
Yes every q-stepper__tab has classes like q-stepper__tab--active or q-stepper__tab--disabled,
you can do something like: 
```
// before
.q-stepper__tab--active .q-stepper__line::before {
  background: green;
}
// after
.q-stepper__tab--active .q-stepper__label::after {
  background: green;
}
```

**Reply by mtn090899** [reply]
This put me on the right track. Thank you!
